docs/docs/cmd/teams/meeting/meeting-create.mdx

```diff
@@ -0,0 +1,241 @@
+                "displayName": null,
+                "tenantId": "ad4f158a-97c7-4914-a9bd-038ecde40ff3",
+                "identityProvider": "AAD"
+              }
+            }
+          }
+        ]
+      },
+      "lobbyBypassSettings": {
+        "scope": "unknownFutureValue",
+        "isDialInBypassEnabled": false
+      },
+      "joinMeetingIdSettings": {
+        "isPasscodeRequired": true,
+        "joinMeetingId": "12345",
+        "passcode": "Z3GYtQ"
+      },
+      "chatInfo": {
+        "threadId": "abc",
+        "messageId": "0",
+        "replyChainMessageId": null
+      },
+      "joinInformation": {
+        "content": "textContent",
+        "contentType": "html"
+      },
+      "watermarkProtection": {
+        "isEnabledForContentSharing": false,
+        "isEnabledForVideo": false
+      }
+    }
+  ]
+  ```
+
+  </TabItem>
+  <TabItem value="Text">
+
+  ```text
+  endDateTime        : 2023-10-05T13:57:09.1889486Z
+  joinUrl            : https://teams.microsoft.com/l/meetup-join/abc
+  recordAutomatically: false
+  startDateTime      : 2023-10-05T12:57:09.1889486Z
+  subject            : null
+  ```
+
+  </TabItem>
+  <TabItem value="CSV">
+
+  ```csv
+  @odata.context,id,creationDateTime,startDateTime,endDateTime,joinUrl,joinWebUrl,meetingCode,isBroadcast,autoAdmittedUsers,isEntryExitAnnounced,allowedPresenters,allowMeetingChat,shareMeetingChatHistoryDefault,allowTeamworkReactions,allowAttendeeToEnableMic,allowAttendeeToEnableCamera,recordAutomatically,allowParticipantsToChangeName,allowRecording,allowTranscription
+https://graph.microsoft.com/v1.0/$metadata#users('id')/onlineMeetings/$entity,abc,2023-10-05T12:56:20.9466495Z,2023-10-05T12:56:20.6594121Z,2023-10-05T13:56:20.6594121Z,https://teams.microsoft.com/l/meetup-join/abc,https://teams.microsoft.com/l/meetup-join/abc,388012423843,,everyoneInCompany,1,everyone,enabled,none,1,1,1,,,1,1
+  ```
+
+  </TabItem>
+  <TabItem value="Markdown">
+
+  ```md
+# teams meeting create --organizerEmail "john.doe@contoso.com"
+
+Date: 05/10/2023
+
+## abc
+
+  Property | Value
+---------|-------
+@odata.context | https://graph.microsoft.com/v1.0/$metadata#users('id')/onlineMeetings/$entity
+id | abc
+creationDateTime | 2023-10-05T12:56:00.6719952Z
+startDateTime | 2023-10-05T12:56:00.2303956Z
+endDateTime | 2023-10-05T13:56:00.2303956Z
+joinUrl | https://teams.microsoft.com/l/meetup-join/abc
+joinWebUrl | https://teams.microsoft.com/l/meetup-join/abc
+meetingCode | 12345
+isBroadcast | false
+autoAdmittedUsers | everyoneInCompany
+isEntryExitAnnounced | true
+allowedPresenters | everyone
+allowMeetingChat | enabled
+shareMeetingChatHistoryDefault | none
+allowTeamworkReactions | true
+allowAttendeeToEnableMic | true
+allowAttendeeToEnableCamera | true
+recordAutomatically | false
+allowParticipantsToChangeName | false
+allowRecording | true
+allowTranscription | true
+  ```
+
+  </TabItem>
+</Tabs>
+
+
+## Remarks
```

Comment:
Let's move this section to right below the options section

Reply:
I hope that it is the correct place. Right after  <Global />